.github/workflows/upgrade-via-tiup.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         path: go/src/github.com/pingcap/dm
 
     - name: Build
-      if: ${{ github.ref != 'refs/heads/main' }}
+      if: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
originally, because DM uses master not main as branch name, all these steps will not be filterd. 🤔 Do you want to skip this action?

Reply:
Yes. If it's master branch, we use nightly version, don't need to build local version